tools/generator/__main__.py

```diff
@@ -22,6 +25,7 @@ def main():
 
     sp = parser.add_subparsers(dest='command', help='Generator to use')
     sp.add_parser('player', help="create player stubs for all languages")
+    sp.add_parser('champions', help="create player stubs for all languages (experimental)")
```

Comment:
The other commands are singular

Reply:
(same in the rest of the code? make it consistent if you choose to use singular)

Reply:
Maybe `playerpython` that we'll rename when it's no longer in beta?

Reply:
Not everything is in singular, rules is plural. But I’m OK for playerpython.

Reply:
Use player / ruby-player instead now to work with gendiff

Reply:
Well if I do that, we’ll only be able to use `stechec2-generator player` for c++ champions.

Reply:
I think we can commit to having all the generators in Jinja for the next finals, so that shouldn't be a problem

Reply:
Ok, we can try that. It is now named player